backend/scripts/seed.sh

```
@@ -1387,63 +1387,93 @@ seed_trip_bali() {
     local p1 p2 p3
     p1=$(insert_pitch_sql "$trip_id" "$my_id" \
-        "The case for Bali in December" \
-        "Dry season runs until October but December is actually still great. Rainy season means lush green landscapes, fewer tourists, and significantly cheaper flights and accommodation. I did the math and we save roughly 40% vs. peak season." 105)
+        "Ubud, Bali" \
+        "The cultural heart of Bali. Rice terraces, temples, yoga retreats, and incredible Balinese cooking. December is rainy season but that means lush green landscapes, fewer tourists, and 40 percent cheaper than peak." 105)
     p2=$(insert_pitch_sql "$trip_id" "$MAYA_ID" \
-        "I found the perfect villa" \
-        "There is a 5-bedroom compound in Ubud with private pool and staff. Split 5 ways it is less than a mid-range hotel. I will drop the Airbnb link in chat." 70)
+        "Seminyak, Bali" \
+        "Beach clubs, rooftop bars, great restaurants, and easy access to Tanah Lot temple. More upscale vibe than Ubud. I found a 5-bedroom villa with private pool that splits to less than a hotel per person." 70)
     p3=$(insert_pitch_sql "$trip_id" "$CARLOS_ID" \
```

Shellcheck (0.11.0)

[warning] p3 appears unused. Verify use (or export if used externally).

(SC2034)